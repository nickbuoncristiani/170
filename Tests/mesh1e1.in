48
25
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48
5 35 8 11 25 26 13 38 20 44 4 12 40 21 9 17 7 19 43 14 3 32 47 29 27
37
x 1.66236 x x 1.07364 1.54275 1.77663 1.04388 1.95933 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.66236 x 1.22408 1.33112 1.6687 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x 1.22408 x 1.61509 x x x x x x x x x x 1.85322 x 1.73852 1.59271 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x 1.33112 1.61509 x 1.31065 x 1.77712 x x x x x x x x x x 1.5925 x x x x x x x x x x x x x x x x x x x x x x x x x 1.03907 x x x 1.16256
1.07364 1.6687 x 1.31065 x x 1.25365 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.54275 x x x x x x x 1.00799 1.92564 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.77663 x x 1.77712 1.25365 x x 1.39825 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.65461 x 1.96716 x x x x x 1.67505
1.04388 x x x x x 1.39825 x 1.03765 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.65004 1.06283 x x x x x x x
1.95933 x x x x 1.00799 x 1.03765 x 1.97426 x 1.15093 x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.5412 x x x x x x x
x x x x x 1.92564 x x 1.97426 x 1.8375 1.99371 1.09261 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x 1.8375 x x 1.33377 1.67811 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x 1.15093 1.99371 x x 1.70529 x x x x x x x x x x x x x x x x x x x 1.93106 1.03582 x x x x x x 1.6937 x x x x x 1.08681 x
x x x x x x x x x 1.09261 1.33377 1.70529 x 1.33818 x 1.34183 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.4392 x
x x x x x x x x x x 1.67811 x 1.33818 x 1.36631 1.61062 1.91446 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x 1.85322 x x x x x x x x x x 1.36631 x x 1.98947 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x 1.34183 1.61062 x x 1.88917 x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.83676 1.38755 1.50182 x
x x 1.73852 x x x x x x x x x x 1.91446 1.98947 1.88917 x 1.72568 x x x x x x x x x x x x x x x x x x x x x x x x x x 1.55929 x x x
x x 1.59271 1.5925 x x x x x x x x x x x x 1.72568 x x x x x x x x x x x x x x x x x x x x x x x x x 1.87851 1.64849 1.25091 x x x
x x x x x x x x x x x x x x x x x x x 1.61019 x x 1.36797 x x 1.76971 1.0405 x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x 1.61019 x 1.1854 1.11039 1.91375 1.11776 1.84309 x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x 1.1854 x 1.35785 x x x x x x x x x x x x x 1.23186 x 1.28952 x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x 1.11039 1.35785 x x x 1.1991 x x x x x x x x x x x x 1.29056 x 1.39484 x 1.41199 x x x x x x
x x x x x x x x x x x x x x x x x x 1.36797 1.91375 x x x 1.40755 x x 1.73704 x x x x x x x x x x x x x x x 1.88265 1.34243 x x x x
x x x x x x x x x x x x x x x x x x x 1.11776 x x 1.40755 x 1.16299 x x x x x x x x x x x x x x x x x x 1.55902 x x x 1.06938
x x x x x x x x x x x x x x x x x x x 1.84309 x 1.1991 x 1.16299 x x x x x x x x x x x x x x x x x 1.42494 x x x x x 1.78015
x x x x x x x x x x x x x x x x x x 1.76971 x x x x x x x 1.55648 1.50696 1.40511 1.66486 1.0862 x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x 1.0405 x x x 1.73704 x x 1.55648 x x 1.32919 x x x x x x x x x x x x x 1.69861 x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x 1.50696 x x x x 1.89336 1.86755 1.78377 1.92734 1.32624 x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x 1.40511 1.32919 x x 1.97952 x x x x x x x x x x x x 1.87989 x 1.35325 x x x
x x x x x x x x x x x x x x x x x x x x x x x x x 1.66486 x x 1.97952 x 1.95339 x x x x x x x x x x x x x 1.83205 1.74417 x x
x x x x x x x x x x x x x x x x x x x x x x x x x 1.0862 x 1.89336 x 1.95339 x x x x 1.50007 x x x x x x x x x x 1.42757 x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.86755 x x x x 1.66519 x x 1.72994 1.66059 x x x x x x x x x x x
x x x x x x x x x x x 1.93106 x x x x x x x x x x x x x x x 1.78377 x x x 1.66519 x 1.68471 x x 1.35391 x x x 1.82199 x x x x x x x
x x x x x x x x x x x 1.03582 x x x x x x x x x x x x x x x 1.92734 x x x x 1.68471 x 1.18762 x x x x x x x x x x x 1.35713 x
x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.32624 x x 1.50007 x x 1.18762 x x x x x x x x x x x 1.08482 1.94323 x
x x x x x x x x x x x x x x x x x x x x 1.23186 x x x x x x x x x x 1.72994 x x x x 1.72185 1.701 1.24785 x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.66059 1.35391 x x 1.72185 x x 1.60923 x 1.22012 x x x x x x x
x x x x x x x x x x x x x x x x x x x x 1.28952 1.29056 x x x x x x x x x x x x x 1.701 x x 1.03814 1.96762 x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.24785 1.60923 1.03814 x 1.79382 1.05826 x x x x x x x
x x x x x x 1.65461 1.65004 x x x x x x x x x x x x x 1.39484 x x x x x x x x x x x x x x x 1.96762 1.79382 x 1.59515 1.6704 x x x x x x
x x x x x x x 1.06283 1.5412 x x 1.6937 x x x x x x x x x x x x x x x x x x x x 1.82199 x x x 1.22012 x 1.05826 1.59515 x x x x x x x x
x x x x x x 1.96716 x x x x x x x x x x x x x x 1.41199 x x 1.42494 x x x x x x x x x x x x x x 1.6704 x x x x x x x 1.56426
x x x x x x x x x x x x x x x x x 1.87851 x x x x 1.88265 x x x 1.69861 x 1.87989 x x x x x x x x x x x x x x 1.77731 1.4048 x x x
x x x 1.03907 x x x x x x x x x x x x x 1.64849 x x x x 1.34243 1.55902 x x x x x x x x x x x x x x x x x x 1.77731 x x x x 1.27234
x x x x x x x x x x x x x x x 1.83676 1.55929 1.25091 x x x x x x x x x x 1.35325 1.83205 x x x x x x x x x x x x 1.4048 x x 1.35144 x x
x x x x x x x x x x x x x x x 1.38755 x x x x x x x x x x x x x 1.74417 1.42757 x x x 1.08482 x x x x x x x x x 1.35144 x 1.6536 x
x x x x x x x x x x x 1.08681 1.4392 x x 1.50182 x x x x x x x x x x x x x x x x x 1.35713 1.94323 x x x x x x x x x x 1.6536 x x
x x x 1.16256 x x 1.67505 x x x x x x x x x x x x x x x x 1.06938 1.78015 x x x x x x x x x x x x x x x x 1.56426 x 1.27234 x x x x
