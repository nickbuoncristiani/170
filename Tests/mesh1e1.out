37 32 33 41 8 7 5 4 3 17 18 44 43 27 19 20 19 26 19 20 22 40 38 21 22 40 8 9 12 13 47 35 28 32 37
21
32 32
8 8
7 7
5 5
4 4
3 3
17 17 14
44 44
43 43
27 27
19 19
20 25 20
26 26 29
40 40
38 38
21 21
9 9
12 12
13 11 13
47 47
35 35
